### ISSUE TITLE
No events, no zoom in/out ==> Time column collapsed

### ISSUE THREAD

**Issue by thelittlefireman** [thellmund/Android-Week-View#106]
**Describe the bug**
if no events, at first start whithout zoom, in or out the time column is collapsed wiredly.

**To Reproduce**
```
<androidx.swiperefreshlayout.widget.SwipeRefreshLayout
        android:id="@+id/swipeToRefreshEdt"
        android:layout_alignParentBottom="true"
        android:layout_height="0dp"
        android:layout_width="match_parent"
        app:layout_constraintBottom_toBottomOf="parent"
        app:layout_constraintTop_toBottomOf="@+id/linearLayoutExpansionpPanel">

        <com.alamkanak.weekview.WeekView
            android:id="@+id/weekView"
            android:layout_alignParentBottom="true"
            android:layout_height="match_parent"
            android:layout_width="match_parent"
            app:columnGap="5dp"
            app:eventTextSize="12sp"
            app:maxHour="20"
            app:eventCornerRadius="5dp"
            app:eventMarginVertical="1dp"
            app:eventTextColor="@color/colorAccent"
            app:minHour="6"
            app:nowLineColor="@color/red"
            app:numberOfVisibleDays="5"
            app:headerRowBackgroundColor="#ffefefef"
            app:headerRowPadding="5dp"
            app:hourHeight="10dp"
            app:showNowLine="true"
            app:timeColumnBackgroundColor="@color/colorPrimaryLight"
            app:timeColumnPadding="5dp"
            app:timeColumnTextColor="@color/colorPrimary"
            app:overlappingEventGap="2dp"
            app:todayBackgroundColor="@color/colorBackgroundMain"
            app:xScrollingSpeed="1" />
    </androidx.swiperefreshlayout.widget.SwipeRefreshLayout>

```

**Expected behavior**
Time column with a good gap and disposition at start

**Screenshots**

![screenschot](https://i.imgur.com/neQ73sw.png)

**Additional context**
 - Android version:  Android 9.0
 - WeekView version: 3.4.2

**Comment by thellmund** [comment]
I added a `SwipeRefreshLayout` around `WeekView` in the sample app and can’t reproduce the issue. Can you provide me with a small sample app that shows the issue?

**Comment by thelittlefireman** [comment]
my bad : it's seems to be the hourHeight xml field 👍

**Comment by thelittlefireman** [comment]
Finaly it's not working, i create a sample that reproduce the problem:

https://github.com/thelittlefireman/IssueWeekView

**Comment by thelittlefireman** [comment]
I try in 3.4.2 same result

**Comment by thellmund** [comment]
Thanks for providing a sample. I’ll have a look soon.

**Comment by thellmund** [comment]
I have identified and fixed the issue in 3cc3804891b81235f9e9cba687bf323b426d81e0. Give it a try and let me know if anything else needs fixing. 

Thanks for reporting!
